java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -306,7 +305,7 @@ public abstract class BaseTransaction<T extends BaseTransaction<T>> {
      * }</pre>
      */
     public T customCommand(String[] args) {
-        ArgsArray commandArgs = buildArgs(args);
+        String[] commandArgs = buildArgs(args);
```

Comment:
Is this on purpose?  I don't know why you changed this.

Reply:
Nevemind - I see the reason now.  This can cause merge conflicts with everyone - we will need to ask everyone to rebase after this is merged.

Reply:
Yeah it's a mess and very slow to fix by hand. I think after Java commands are done we probably want to robocode this.